tests/functional/test_import_data_bkp/config_generator_negative.py

```diff
@@ -79,13 +79,21 @@
 """
 
 for variable in VARIABLES:
-    d_name = "service_import_check_negative/{}_{}_{}".format(variable[0], variable[3], variable[4])
+    d_name = "service_import_check_negative/{}_{}_{}".format(variable[0],
+                                                             variable[3],
+                                                             variable[4])
```

Comment:
```suggestion
    d_name = "service_import_check_negative/{}_{}_{}".format(
        variable[0], variable[3], variable[4]
    )
```
Black-like formatting is more preferable